src/watertap_contrib/reflo/analysis/multiperiod/ltmed_vagmd_semibatch/test/test_MED_VAGMD_semibatch_flowsheet_multiperiod.py

```diff
@@ -0,0 +1,146 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pytest
+from pyomo.environ import (
+    ConcreteModel,
+    assert_optimal_termination,
+    units as pyunits,
+)
+
+from watertap_contrib.reflo.analysis.multiperiod.ltmed_vagmd_semibatch.MED_VAGMD_semibatch_class import (
+    MEDVAGMDsemibatch,
+)
+
+from idaes.core import FlowsheetBlock
+from idaes.core.solvers import get_solver
+
+from idaes.core.util.model_statistics import (
+    degrees_of_freedom,
+)
+
+solver = get_solver()
+
+
+class TestVAGMDbatch:
+    @pytest.fixture(scope="class")
+    def MED_VAGMD_semibatch_frame(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(dynamic=False)
+        model_input = {
+            "n_time_points": 90,
+            "med_feed_salinity": 35,
+            "med_feed_temp": 25,
+            "med_steam_temp": 80,
+            "med_capacity": 1.5,
+            "med_recovry_ratio": 0.5,
+            "md_feed_flow_rate": 600,
+            "md_evap_inlet_temp": 80,
+            "md_cond_inlet_temp": 25,
+            "md_module_type": "AS26C7.2L",
+            "md_cooling_system_type": "closed",
+            "md_cooling_inlet_temp": 25,
+            "md_high_brine_salinity": False,
+            "dt": 60,
+            "batch_volume": 50,
+        }
```

Comment:
There should be some testing for when the user does not provide the input correctly to make sure we're catching issues and raising exceptions that would be helpful.

Reply:
Sure! Testing for inputs check is added now.